pyof/v0x01/asynchronous/error_msg.py

```diff
@@ -162,7 +161,7 @@ class ErrorMsg(GenericMessage):
     header = Header(message_type=Type.OFPT_ERROR)
     error_type = UBInt16(enum_ref=ErrorType)
     code = UBInt16()
-    data = ConstantTypeList()
+    data = BinaryData()
 
     def __init__(self, xid=None, error_type=None, code=None, data=None):
         """Assign parameters to object attributes.
```

Comment:
The *data* attribute can be empty right? So, I think it is better to set the default value to **b""** instead of **None**.